lib/turn.rb

```diff
@@ -0,0 +1,21 @@
+class Turn
+	attr_reader :guess, :card
+
+	def initialize(guess, card)
+		@guess = guess
+		@card = card
+	end
+
+	def correct?
+		@guess == card.answer
+	end
+
+	def feedback
+		if correct? == true
+		"Correct!"
+		elsif correct? == false
+		"Incorrect."
+		end
+	end
```

Comment:
since `correct?` evaluates to true or false automatically, you could slim up this method a bit by doing something like: 

```
	def feedback
		if correct?
		"Correct!"
		else
		"Incorrect."
		end
	end
```